doc/integrator/ngeo.rst

```diff
@@ -108,3 +108,23 @@ The map configuration will be here:
    If you previously had a CGXP application and want to keep your permalinks
    compatible, the resolutions should be the same as in the previous
    application.
+
+Dynamic.js view
+---------------
+
+To configure the ngeo constants with dynamic or configurable values we have the dynamic view.
+
+This view is configurable in the vars files in the section ``interfaces_config``.
+The sub section is the interface name, and after we have:
+
+* ``redirect_interface``: interface on witch one we can be redirected if we use the wrong type of device
+  (mobile/desktop).
+* ``do_redirect``: directly do the redirect.
+* ``constants``: Directly define an ``angular`` constant in the vars file.
+* ``routes``: key: constant name, value: name of the route witch one we want to have the URL.
+* ``static``: key: constant name, value: name of the static view that we want to have the URL.
+* ``fulltextsearch_params``: Additional parameters to the full text search view, see
+  :ref:`developer_webservices_fts` for more information.
+* ``wfs_permalink``: Additional values for the ``ngeoWfsPermalinkOptions`` constant, see:
```

Comment:
ngeoWfsPermalinkOptions is not a constant, but something else, I guess?

Reply:
it's an angular constant name...

Reply:
But isn't it strange to provide additional values for a constant? (constant = a constant value to me, not something where you can provide additional values)

Reply:
In code I do that (in mako template):
```
<%
wfs_permalink = dict(url=request.route_url('mapserverproxy'))
wfs_permalink.update(interface_config.get('wfs_permalink', {}))
%>
module.constant('ngeoWfsPermalinkOptions', ${json.dumps(wfs_permalink) | n});
```
...

Reply:
OK, I understand, thanks.